examples/alert_stream_integration_endpoint/aiokafka_consumer/aiokafka_consumer.py

```diff
@@ -62,24 +59,19 @@ async def main():
         # These next two properties tell the Kafka client about the specific
         # authentication and authorization protocols that should be used when
         # connecting.
-        security_protocol="SASL_SSL",
+        ssl_context=None,
+        security_protocol="SASL_PLAINTEXT",
```

Comment:
why did we switch to plaintext?

Reply:
We do not have encryption set up on the kafka cluster. We can do it, but it was discussed that the data does not need to be decrypted so there isn't a need to put that layer of encryption on.